go/ql/src/experimental/CWE-942/CorsMisconfiguration.ql

```diff
@@ -93,10 +119,13 @@ predicate allowCredentialsIsSetToTrue(AllowOriginHeaderWrite allowOriginHW) {
  * UntrustedFlowSource.
  * The `message` parameter is populated with the warning message to be returned by the query.
  */
-predicate flowsFromUntrustedToAllowOrigin(AllowOriginHeaderWrite allowOriginHW, string message) {
+predicate flowsFromUntrustedToAllowOrigin(DataFlow::ExprNode allowOriginHW, string message) {
```

Comment:
@owen-mc I will leave the name as is since it seems like allowOriginHW is used all parts of the query, and I don't think changing for all is desired. Additionally, the cors gin framework will eventually write all the values to the header using the middleware. If you would still like to me change it, just let me know.